mule-migration-tool-e2e-tests/src/test/resources/e2e/gaps/unknown_namespace/output/report/report.json

```diff
@@ -38,22 +38,24 @@
     {
       "level": "ERROR",
       "key": "components.unsupported",
-      "component": "unknown:source",
-      "lineNumber": 5,
-      "columnNumber": 25,
-      "message": "The migration of \u0027unknown\u0027 is not supported.",
+      "component": "flow",
+      "lineNumber": 4,
+      "columnNumber": 36,
+      "message": "The migration of 'namespace unknown (element operation)' is not supported.",
       "filePath": "src/main/mule/mule-config.xml",
-      "documentationLinks": []
+      "documentationLinks":
+      []
     },
     {
       "level": "ERROR",
       "key": "components.unsupported",
-      "component": "unknown:operation",
-      "lineNumber": 16,
-      "columnNumber": 28,
-      "message": "The migration of \u0027unknown\u0027 is not supported.",
+      "component": "flow",
```

Comment:
should report `unknown:operation` as the error component with its line and col numbers